File: src/test/java/no/nav/boot/conditionals/ClusterTest.java
package no.nav.boot.conditionals;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.extension.ExtendWith;
import org.springframework.test.context.junit.jupiter.SpringExtension;

@ExtendWith(SpringExtension.class)
class ClusterTest {

    @Test
    void metoderBruktITestEllerDevMåIkkeInneholderProdCluster() {
        assertThat(Cluster.devClusters()).doesNotContain(Cluster.PROD_FSS, Cluster.PROD_SBS, Cluster.PROD_FSS);
        assertThat(Cluster.notProdClusters()).doesNotContain(Cluster.PROD_FSS, Cluster.PROD_SBS, Cluster.PROD_FSS);
        assertThat(Cluster.localOrTest()).doesNotContain(Cluster.PROD_FSS, Cluster.PROD_SBS, Cluster.PROD_FSS);
    }

}

File: src/test/kotlin/no/nav/boot/conditionals/ClusterTest.kt
package no.nav.boot.conditionals

import no.nav.boot.conditionals.Cluster.Companion.devClusters
import no.nav.boot.conditionals.Cluster.Companion.localOrTest
import no.nav.boot.conditionals.Cluster.Companion.notProdClusters
import no.nav.boot.conditionals.Cluster.PROD_FSS
import no.nav.boot.conditionals.Cluster.PROD_SBS
import org.assertj.core.api.Assertions
import org.assertj.core.api.Assertions.*
import org.junit.jupiter.api.Test
import org.junit.jupiter.api.extension.ExtendWith
import org.springframework.test.context.junit.jupiter.SpringExtension

@ExtendWith(SpringExtension::class)
internal class ClusterTest {
    @Test
    fun metoderBruktITestEllerDevMåIkkeInneholderProdCluster() {
        assertThat(devClusters()).doesNotContain(PROD_FSS, PROD_SBS, PROD_FSS)
        assertThat(notProdClusters()).doesNotContain(PROD_FSS, PROD_SBS, PROD_FSS)
        assertThat(localOrTest()).doesNotContain(PROD_FSS, PROD_SBS, PROD_FSS)
    }
}